### PR TITLE
Upgrade to later `analyzer` package, which no longer support `isUndefined`

### LINKED ISSUE #55: Generating build_script fails in Flutter channel Dev (1.10.7)
## Steps to Reproduce

1. Create a new application with the default template `flutter create example`
2. Add functional_widget and functional_widget_annotation to pubspec.yaml
```
dependencies:
  flutter:
    sdk: flutter
  functional_widget_annotation: ^0.5.1

builders:
  functional_widget: ^0.7.1
```
3. Run the application. It will fail with: `Error generating build_script snapshot: Could not find a file named "pubspec.yaml" in "/Users/gregorysech/.pub-cache/hosted/pub.dartlang.org/functional_widget-0.7.1".
`

## Logs

```
[ +288 ms] generating build script... (completed in 11.9s)
[   +3 ms] "flutter run" took 16,008ms.
Error generating build_script snapshot: Could not find a file named "pubspec.yaml" in "/Users/gregorysech/.pub-cache/hosted/pub.dartlang.org/functional_widget-0.7.1".



#0      throwToolExit (package:flutter_tools/src/base/common.dart:28:3)
#1      BuildRunner.generateBuildScript (package:flutter_tools/src/build_runner/build_runner.dart:119:9)
<asynchronous suspension>
#2      BuildRunner.daemon (package:flutter_tools/src/build_runner/build_runner.dart:146:11)
<asynchronous suspension>
#3      CodeGeneratingResidentCompiler.create (package:flutter_tools/src/codegen.dart:188:61)
<asynchronous suspension>
#4      FlutterDevice.create (package:flutter_tools/src/resident_runner.dart:68:56)
<asynchronous suspension>
#5      RunCommand.runCommand (package:flutter_tools/src/commands/run.dart:405:29)
<asynchronous suspension>
#6      FlutterCommand.verifyThenRunCommand (package:flutter_tools/src/runner/flutter_command.dart:553:18)
#7      _asyncThenWrapperHelper.<anonymous closure> (dart:async-patch/async_patch.dart:71:64)
#8      _rootRunUnary (dart:async/zone.dart:1132:38)
#9      _CustomZone.runUnary (dart:async/zone.dart:1029:19)
#10     _FutureListener.handleValue (dart:async/future_impl.dart:137:18)
#11     Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:678:45)
#12     Future._propagateToListeners (dart:async/future_impl.dart:707:32)
#13     Future._completeWithValue (dart:async/future_impl.dart:522:5)
#14     _AsyncAwaitCompleter.complete (dart:async-patch/async_patch.dart:30:15)
#15     _completeOnAsyncReturn (dart:async-patch/async_patch.dart:288:13)
#16     RunCommand.usageValues (package:flutter_tools/src/commands/run.dart)
#17     _asyncThenWrapperHelper.<anonymous closure> (dart:async-patch/async_patch.dart:71:64)
#18     _rootRunUnary (dart:async/zone.dart:1132:38)
#19     _CustomZone.runUnary (dart:async/zone.dart:1029:19)
#20     _FutureListener.handleValue (dart:async/future_impl.dart:137:18)
#21     Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:678:45)
#22     Future._propagateToListeners (dart:async/future_impl.dart:707:32)
#23     Future._completeWithValue (dart:async/future_impl.dart:522:5)
#24     _AsyncAwaitCompleter.complete (dart:async-patch/async_patch.dart:30:15)
#25     _completeOnAsyncReturn (dart:async-patch/async_patch.dart:288:13)
#26     IosProject.isSwift (package:flutter_tools/src/project.dart)
#27     _asyncThenWrapperHelper.<anonymous closure> (dart:async-patch/async_patch.dart:71:64)
#28     _rootRunUnary (dart:async/zone.dart:1132:38)
#29     _CustomZone.runUnary (dart:async/zone.dart:1029:19)
#30     _FutureListener.handleValue (dart:async/future_impl.dart:137:18)
#31     Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:678:45)
#32     Future._propagateToListeners (dart:async/future_impl.dart:707:32)
#33     Future._completeWithValue (dart:async/future_impl.dart:522:5)
#34     _AsyncAwaitCompleter.complete (dart:async-patch/async_patch.dart:30:15)
#35     _completeOnAsyncReturn (dart:async-patch/async_patch.dart:288:13)
#36     IosProject.buildSettings (package:flutter_tools/src/project.dart)
#37     _asyncThenWrapperHelper.<anonymous closure> (dart:async-patch/async_patch.dart:71:64)
#38     _rootRunUnary (dart:async/zone.dart:1132:38)
#39     _CustomZone.runUnary (dart:async/zone.dart:1029:19)
#40     _FutureListener.handleValue (dart:async/future_impl.dart:137:18)
#41     Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:678:45)
#42     Future._propagateToListeners (dart:async/future_impl.dart:707:32)
#43     Future._completeWithValue (dart:async/future_impl.dart:522:5)
#44     _AsyncAwaitCompleter.complete (dart:async-patch/async_patch.dart:30:15)
#45     _completeOnAsyncReturn (dart:async-patch/async_patch.dart:288:13)
#46     XcodeProjectInterpreter.getBuildSettings (package:flutter_tools/src/ios/xcodeproj.dart)
#47     _asyncThenWrapperHelper.<anonymous closure> (dart:async-patch/async_patch.dart:71:64)
#48     _rootRunUnary (dart:async/zone.dart:1132:38)
#49     _CustomZone.runUnary (dart:async/zone.dart:1029:19)
#50     _FutureListener.handleValue (dart:async/future_impl.dart:137:18)
#51     Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:678:45)
#52     Future._propagateToListeners (dart:async/future_impl.dart:707:32)
#53     Future._completeWithValue (dart:async/future_impl.dart:522:5)
#54     _AsyncAwaitCompleter.complete (dart:async-patch/async_patch.dart:30:15)
#55     _completeOnAsyncReturn (dart:async-patch/async_patch.dart:288:13)
#56     _DefaultProcessUtils.run (package:flutter_tools/src/base/process.dart)
#57     _asyncThenWrapperHelper.<anonymous closure> (dart:async-patch/async_patch.dart:71:64)
#58     _rootRunUnary (dart:async/zone.dart:1132:38)
#59     _CustomZone.runUnary (dart:async/zone.dart:1029:19)
#60     _FutureListener.handleValue (dart:async/future_impl.dart:137:18)
#61     Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:678:45)
#62     Future._propagateToListeners (dart:async/future_impl.dart:707:32)
#63     Future._completeWithValue (dart:async/future_impl.dart:522:5)
#64     Future.wait.<anonymous closure> (dart:async/future.dart:400:22)
#65     _rootRunUnary (dart:async/zone.dart:1132:38)
#66     _CustomZone.runUnary (dart:async/zone.dart:1029:19)
#67     _FutureListener.handleValue (dart:async/future_impl.dart:137:18)
#68     Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:678:45)
#69     Future._propagateToListeners (dart:async/future_impl.dart:707:32)
#70     Future._complete (dart:async/future_impl.dart:512:7)
#71     _BufferingStreamSubscription.asFuture.<anonymous closure> (dart:async/stream_impl.dart:204:14)
#72     _rootRun (dart:async/zone.dart:1120:38)
#73     _CustomZone.run (dart:async/zone.dart:1021:19)
#74     _CustomZone.runGuarded (dart:async/zone.dart:923:7)
#75     _BufferingStreamSubscription._sendDone.sendDone (dart:async/stream_impl.dart:389:13)
#76     _BufferingStreamSubscription._sendDone (dart:async/stream_impl.dart:399:15)
#77     _BufferingStreamSubscription._close (dart:async/stream_impl.dart:283:7)
#78     _SinkTransformerStreamSubscription._close (dart:async/stream_transformers.dart:96:11)
#79     _EventSinkWrapper.close (dart:async/stream_transformers.dart:23:11)
#80     _StringAdapterSink.close (dart:convert/string_conversion.dart:249:11)
#81     _Utf8ConversionSink.close (dart:convert/string_conversion.dart:300:20)
#82     _ConverterStreamEventSink.close (dart:convert/chunked_conversion.dart:80:18)
#83     _SinkTransformerStreamSubscription._handleDone (dart:async/stream_transformers.dart:141:24)
#84     _rootRun (dart:async/zone.dart:1120:38)
#85     _CustomZone.run (dart:async/zone.dart:1021:19)
#86     _CustomZone.runGuarded (dart:async/zone.dart:923:7)
#87     _BufferingStreamSubscription._sendDone.sendDone (dart:async/stream_impl.dart:389:13)
#88     _BufferingStreamSubscription._sendDone (dart:async/stream_impl.dart:399:15)
#89     _BufferingStreamSubscription._close (dart:async/stream_impl.dart:283:7)
#90     _SyncStreamControllerDispatch._sendDone (dart:async/stream_controller.dart:772:19)
#91     _StreamController._closeUnchecked (dart:async/stream_controller.dart:629:7)
#92     _StreamController.close (dart:async/stream_controller.dart:622:5)
#93     _Socket._onData (dart:io-patch/socket_patch.dart:1836:21)
#94     _rootRunUnary (dart:async/zone.dart:1136:13)
#95     _CustomZone.runUnary (dart:async/zone.dart:1029:19)
#96     _CustomZone.runUnaryGuarded (dart:async/zone.dart:931:7)
#97     _BufferingStreamSubscription._sendData (dart:async/stream_impl.dart:336:11)
#98     _BufferingStreamSubscription._add (dart:async/stream_impl.dart:263:7)
#99     _SyncStreamControllerDispatch._sendData (dart:async/stream_controller.dart:764:19)
#100    _StreamController._add (dart:async/stream_controller.dart:640:7)
#101    _StreamController.add (dart:async/stream_controller.dart:586:5)
#102    new _RawSocket.<anonymous closure> (dart:io-patch/socket_patch.dart:1384:35)
#103    _NativeSocket.issueReadEvent.issue (dart:io-patch/socket_patch.dart:890:18)
#104    _microtaskLoop (dart:async/schedule_microtask.dart:41:21)
#105    _startMicrotaskLoop (dart:async/schedule_microtask.dart:50:5)
#106    _runPendingImmediateCallback (dart:isolate-patch/isolate_patch.dart:116:13)
#107    _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:173:5)
```

```
Analyzing example...                                                    
No issues found! (ran in 3.4s)
```

In stable `flutter run` works as intended but in dev it does not.
```
[✓] Flutter (Channel dev, v1.10.7, on Mac OS X 10.14.6 18G87, locale en-GB)
    • Flutter version 1.10.7 at /Users/gregorysech/flutter
    • Framework revision e70236e36c (27 hours ago), 2019-10-02 09:32:30 -0700
    • Engine revision 9e6314d348
    • Dart version 2.6.0 (build 2.6.0-dev.0.0 1103600280)

 
[✓] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
    • Android SDK at /Users/gregorysech/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-29, build-tools 28.0.3
    • Java binary at: /Volumes/SD/IDEs/apps/AndroidStudio/ch-0/191.5791312/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 10.2.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 10.2.1, Build version 10E1001
    • CocoaPods version 1.7.5

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 3.5)
    • Android Studio at /Volumes/SD/IDEs/apps/AndroidStudio/ch-0/191.5791312/Android Studio.app/Contents
    • Flutter plugin version 39.0.3
    • Dart plugin version 191.8423
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)

[✓] VS Code (version 1.38.1)
    • VS Code at /Users/gregorysech/Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.4.1

[✓] Connected device (3 available)
    • iPhone Xʀ       • AE87AF8A-D548-4345-BE9A-81D64DDF453A • ios            • com.apple.CoreSimulator.SimRuntime.iOS-12-2 (simulator)
    • Chrome          • chrome                               • web-javascript • Google Chrome 77.0.3865.90
    • Headless Server • headless-server                      • web-javascript • Flutter Tools

• No issues found!

```
It works on Stable
```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, v1.9.1+hotfix.4, on Mac OS X 10.14.6 18G87, locale en-GB)
 
[✓] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
[✓] Xcode - develop for iOS and macOS (Xcode 10.2.1)
[✓] Android Studio (version 3.5)
[✓] VS Code (version 1.38.1)
[✓] Connected device (1 available)

• No issues found!
Gregorys-MBP:example gregorysech$ flutter doctor -v
[✓] Flutter (Channel stable, v1.9.1+hotfix.4, on Mac OS X 10.14.6 18G87, locale en-GB)
    • Flutter version 1.9.1+hotfix.4 at /Users/gregorysech/flutter
    • Framework revision cc949a8e8b (6 days ago), 2019-09-27 15:04:59 -0700
    • Engine revision b863200c37
    • Dart version 2.5.0

 
[✓] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
    • Android SDK at /Users/gregorysech/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-29, build-tools 28.0.3
    • Java binary at: /Volumes/SD/IDEs/apps/AndroidStudio/ch-0/191.5791312/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 10.2.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 10.2.1, Build version 10E1001
    • CocoaPods version 1.7.5

[✓] Android Studio (version 3.5)
    • Android Studio at /Volumes/SD/IDEs/apps/AndroidStudio/ch-0/191.5791312/Android Studio.app/Contents
    • Flutter plugin version 39.0.3
    • Dart plugin version 191.8423
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)

[✓] VS Code (version 1.38.1)
    • VS Code at /Users/gregorysech/Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.4.1

[✓] Connected device (1 available)
    • iPhone Xʀ • AE87AF8A-D548-4345-BE9A-81D64DDF453A • ios • com.apple.CoreSimulator.SimRuntime.iOS-12-2 (simulator)

• No issues found!

```
# Notes
I'm adding this issue here as I am not sure if it's a package problem or a build system problem. 
If needed I can add this issue on the Flutter repository.

### LINKED ISSUE #63: Generating build_script fails in Flutter 1.12.13
I guess this issue is same with https://github.com/rrousselGit/functional_widget/issues/55#issue-502262720,
but my sdk is stable channel v1.12.13+hotfix.8

try with:
functional_widget_annotation: ^0.5.1
functional_widget: ^0.7.2

when i run the app, error occurs.
the output is:
`generating build script...                                         31.9s
Exception: Error generating build_script snapshot: ../../../base/flutterSDK/flutter/.pub-cache/hosted/pub.flutter-io.cn/functional_widget-0.7.2/lib/function_to_widget_class.dart:216:43: Error: The getter 'isUndefined' isn't defined for the class 'DartType'.
 - 'DartType' is from 'package:analyzer/dart/element/type.dart' ('../../../base/flutterSDK/flutter/.pub-cache/hosted/pub.flutter-io.cn/analyzer-0.39.4/lib/dart/element/type.dart').
Try correcting the name to the name of an existing getter, or defining a getter or field named 'isUndefined'.
      'DiagnosticsProperty<${element.type.isUndefined ? findBeginToken(element) : element.type.displayName}>';
                                          ^^^^^^^^^^^
../../../base/flutterSDK/flutter/.pub-cache/hosted/pub.flutter-io.cn/functional_widget-0.7.2/lib/src/parameters.dart:57:20: Error: The getter 'isUndefined' isn't defined for the class 'DartType'.
 - 'DartType' is from 'package:analyzer/dart/element/type.dart' ('../../../base/flutterSDK/flutter/.pub-cache/hosted/pub.flutter-io.cn/analyzer-0.39.4/lib/dart/element/type.dart').
Try correcting the name to the name of an existing getter, or defining a getter or field named 'isUndefined'.
  if (element.type.isUndefined) {
                   ^^^^^^^^^^^`
after i remove the dependecy of functional_widget_annotation and functional_widget, i can run the app successfully.
PS: Mac 10.14.6 and Android Studio 3.6

### LINKED ISSUE #64: Cannot run "flutter pub pub run build_runner watch" on Flutter dev channel
I am using **functional_widget** for a long time (on both dev and master flutter dev channels).

After upgrading flutter dev channel last week, ```functional_widget-0.7.2``` stops working (0.7.1 works well):

```
flutter pub pub run build_runner watch
[INFO] Generating build script...
[INFO] Generating build script completed, took 604ms

[INFO] Creating build script snapshot......
[INFO] Creating build script snapshot... completed, took 7.5s

[SEVERE] Failed to snapshot build script .dart_tool/build/entrypoint/build.dart.
This is likely caused by a misconfigured builder definition.
[SEVERE] /C:/Users/pzika/AppData/Roaming/Pub/Cache/hosted/pub.dartlang.org/functional_widget-0.7.2/lib/function_to_widget_class.dart:216:43: Error: The getter 'isUndefined' isn't defined for the class 'DartType'. - 'DartType' is from 'package:analyzer/dart/element/type.dart' ('/C:/Users/pzika/AppData/Roaming/Pub/Cache/hosted/pub.dartlang.org/analyzer-0.39.4/lib/dart/element/type.dart').Try correcting the name to the name of an existing getter, or defining a getter or field named 'isUndefined'.      'DiagnosticsProperty<${element.type.isUndefined ? findBeginToken(element) : element.type.displayName}>';                                          ^^^^^^^^^^^/C:/Users/pzika/AppData/Roaming/Pub/Cache/hosted/pub.dartlang.org/functional_widget-0.7.2/lib/src/parameters.dart:57:20: Error: The gettePS 

C:\wikibulary\dart\know_how> flutter doctor
Doctor summary (to see all details, run flutter doctor -v):
[√] Flutter (Channel dev, v1.15.3, on Microsoft Windows [Version 10.0.18362.657], locale cs-CZ)
[!] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
    X No Java Development Kit (JDK) found; You must have the environment variable JAVA_HOME set and the java
      binary in your PATH. You can download the JDK from
      https://www.oracle.com/technetwork/java/javase/downloads/.
[√] Chrome - develop for the web
[!] Android Studio (version 3.5)
    X Flutter plugin not installed; this adds Flutter specific functionality.
    X Dart plugin not installed; this adds Dart specific functionality.
    X Unable to determine bundled Java version.
[√] VS Code (version 1.42.1)
[√] Connected device (2 available)

! Doctor found issues in 2 categories.
PS C:\wikibulary\dart\know_how>
```

### PR DESCRIPTION
**Pull request by truongsinh**
Close #55 Close #63 Close #64

I added several more unit test, one of which seems to be functionally incorrect, but at least it guards against the crash (due to null `element.session`)

### CODE REVIEW

File: functional_widget/test/diagnostics_test.dart [hunk 1]
```diff
@@ -125,6 +125,27 @@ class FunctionTest extends StatelessWidget {
...
+      // `ObjectFlagProperty<dynamic>.has`
+      await _expect('typedefTest', completion('''
+class TypedefTest<T> extends StatelessWidget {
+  const TypedefTest(this.a, {Key key}) : super(key: key);
+
+  final void Function(dynamic) a;
+
+  @override
+  Widget build(BuildContext _context) => typedefTest<T>(a);
+  @override
```
Comment (truongsinh) [inline functional_widget/test/diagnostics_test.dart:148]: functionally seems to be suboptimal, but at least for now it's still working for developer, and not crash due to https://github.com/rrousselGit/functional_widget/pull/67/files#diff-8725693a4d1b56a9253011b227f468ddL7 `element.session` being `null`.

**Comment by truongsinh** [comment]
I really hate rounding and floating point number 😒

File: code_gen_tester/pubspec.yaml [hunk 2]
```diff
@@ -1,13 +1,13 @@
 name: code_gen_tester
 version: 0.0.1
 environment:
-    sdk: '>=2.0.0-dev.68.0 <3.0.0'
+    sdk: '>=2.1.0 <3.0.0'
 dependencies:
     source_gen: ^0.9.4+1
     build: '>=1.1.2 <2.0.0'
-    analyzer: '>=0.35.4 <0.36.0'
+    analyzer: '>=0.35.4 <1.0.0'
```
Comment (rrousselGit) [inline code_gen_tester/pubspec.yaml:8]:
```suggestion
    analyzer: '>=0.39.0 <1.0.0'
```

File: code_gen_tester/pubspec.yaml [hunk 3]
```diff
@@ -1,13 +1,13 @@
 name: code_gen_tester
 version: 0.0.1
 environment:
-    sdk: '>=2.0.0-dev.68.0 <3.0.0'
+    sdk: '>=2.1.0 <3.0.0'
 dependencies:
```
Comment (rrousselGit) [inline code_gen_tester/pubspec.yaml:5]: nit: do you think you could upgrade the other dependencies too? like build_runner

It should be straightforward. But if it's not, don't worry too much about it.

File: functional_widget/lib/findBeginToken.dart [hunk 4]
```diff
@@ -1,16 +1,13 @@
 import 'package:analyzer/dart/ast/ast.dart';
-import 'package:analyzer/dart/ast/token.dart';
 import 'package:analyzer/dart/element/element.dart';
 
-Token findBeginToken(ParameterElement element) {
+String findBeginToken(ParameterElement element) {
```
Comment (rrousselGit) [inline functional_widget/lib/findBeginToken.dart:4]: The function should be renamed, as it doesn't return a Token anymore

May I suggest `tryParseUndefinedType`?

File: functional_widget/pubspec.yaml [hunk 5]
```diff
@@ -19,7 +19,7 @@ dependencies:
 dev_dependencies:
```
Comment (rrousselGit) [inline functional_widget/pubspec.yaml:19]: the `dependencies:` needs to be upgraded here too, especially `analyzer`

File: functional_widget/test/diagnostics_test.dart [hunk 6]
```diff
@@ -125,6 +125,27 @@ class FunctionTest extends StatelessWidget {
...
   @override
+  void debugFillProperties(DiagnosticPropertiesBuilder properties) {
+    super.debugFillProperties(properties);
+    properties.add(DiagnosticsProperty<void Function()>('a', a));
+  }
+}
+'''));
+    });
+    test('typedef type', () async {
+      // @todo should be `final void Function(T) a;` instead of
```
Comment (rrousselGit) [inline functional_widget/test/diagnostics_test.dart:136]:
```suggestion
      // TODO should be `final void Function(T) a;` instead of
```

File: code_gen_tester/pubspec.yaml (hunk 2 above)
Comment (truongsinh) [reply code_gen_tester/pubspec.yaml:8]: Do you really want to go that high immediately? My original intention is to have range quite big to be a little bit "backward compatible", but I'm fine going godspeed. And if so, there's another deprecated field (`displayName`) that I will upgrade as well.

File: functional_widget/lib/findBeginToken.dart (hunk 4 above)
Comment (truongsinh) [reply functional_widget/lib/findBeginToken.dart:4]: oh yeah true, but I'd think it's more like `tryParseDynamicType`

File: code_gen_tester/pubspec.yaml (hunk 3 above)
Comment (truongsinh) [reply code_gen_tester/pubspec.yaml:5]: I'm fine doing that, but referring to my other comment, I want to discuss about latest dependency vs backward compatibility for other developers.

File: code_gen_tester/pubspec.yaml (hunk 2 above)
Comment (rrousselGit) [reply code_gen_tester/pubspec.yaml:8]: I don't really have the time to maintain a wide range of versions, so yes

File: code_gen_tester/pubspec.yaml (hunk 3 above)
Comment (rrousselGit) [reply code_gen_tester/pubspec.yaml:5]: code_gen_tester is internal to functional_widget so that shouldn't matter in this specific case

**Comment by truongsinh** [comment]
@rrousselGit done upgrading dependencies, though brace for big changes 😂

**Comment by rrousselGit** [comment]
Haha thanks again for working on that!
You're saving me a lot of trouble :heart:

**Comment by rrousselGit** [comment]
Looks good to me, thanks again!
